libbeat/docs/loggingconfig.asciidoc

```diff
@@ -43,12 +43,12 @@ You can specify the following options in the `logging` section of the +{beatname
 
 ===== to_syslog
 
-Sends all logging output to syslog if enabled. For non-Windows systems, the default
-value is true.
+If enabled, sends all logging output to syslog (on non-Windows systems only). The default
```

Comment:
Just a late follow-up with some ancillary info, the Golang [syslog](https://golang.org/src/log/syslog/syslog.go) package isn't implemented on Windows. It has a `// +build !windows` build tag.